api/src/main/java/org/openmrs/module/appointmentschedulingui/reporting/library/AppointmentSchedulingUIDataSetDefinitionLibrary.java

```diff
@@ -102,6 +103,7 @@ public AppointmentDataSetDefinition getDailyAppointmentsDataSetDefinition() {
         dsd.addColumn("startDatetime", new AppointmentStartDateDataDefinition(), "", null);
         dsd.addColumn("startTimeFormatted", new AppointmentStartDateDataDefinition(), "", new DateConverter(AppointmentSchedulingUIConstants.TIME_FORMAT));
         dsd.addColumn("endTimeFormatted", new AppointmentEndDateDataDefinition(), "", new DateConverter(AppointmentSchedulingUIConstants.TIME_FORMAT));
+        dsd.addColumn("creator", new AppointmentCreatorDataDefinition(), "", formatted);
```

Comment:
This is where you would add the appropriate converters @brandones, if the AppointmentCreatorDataDefinition is changed to return a User, then you would add 1 or more converters here, chained together, to convert from the User to whatever String representation you want to output.